js/views/modals/About/Donations.js

```diff
@@ -16,34 +18,27 @@ export default class extends baseVw {
 
   events() {
     return {
-      'click #copyAddress': 'copyDonationAddress',
-      'click #donationAddress': 'copyDonationAddress',
+      'click .js-copyAddress': 'copyDonationAddress',
     };
   }
 
   copyDonationAddress() {
-    const address = document.querySelector('#donationAddress');
-    const range = document.createRange();
-    const selection = window.getSelection();
-    range.selectNodeContents(address);
-    selection.removeAllRanges();
-    selection.addRange(range);
-
     clipboard.writeText(obDonationAddress);
 
-    this.$('#copyNotification').addClass('active');
+    this.$('.copyNotification').addClass('active');
     if (!!hiderTimer) {
       clearTimeout(hiderTimer);
     }
     hiderTimer = setTimeout(
-      () => this.$('#copyNotification').removeClass('active'), 3000);
+      () => this.$('.copyNotification').removeClass('active'), 3000);
```

Comment:
Can we please cache the `.copyNotification` element, rather than querying the DOM multiple times for it?

Since it's only being used in this method, it could be as simple as:

```
const $copyNotification = this.$('.copyNotification').addClass('active');
```

then in the timeout:

```
() => $copyNotification.removeClass('active'), 3000);
```

If you think that element will be used in other parts of the view and you want to get real fancy, you could take this approach:

in render - 

```
this._$copyNotification = null; // prefaced with an underscore
```

separate getter function -

```
  get $copyNotification() {
    return this._$copyNotification ||
      (this._$copyNotification = this.$('.copyNotification'));
  }
```

This approach has the benefit in that it will only query the DOM when and if the element is needed and it would make it available view wide. It's an approach we take in most of our other views. Of course, it only really makes sense if an element will conditionally be necessary depending on the code path a user takes. If the element will always be necessary and potentially be necessary more than once, then we could just cache in render (e.g. `$myEl = $('.js-myEl')`).